src/static/riot/datasets/management.tag

```diff
@@ -52,6 +57,12 @@
                     <i class="icon delete"></i>
                 </button>
             </td>
+            <td class="center aligned">
+                <div class="ui fitted checkbox">
+                    <input type="checkbox" name="delete_checkbox" onclick="{ mark_dataset_for_deletion.bind(this, dataset) }">
+                    <label></label>
```

Comment:
label needed?

Reply:
I believe the checkboxes disappear if the label tag is deleted. Like this: 
![image](https://user-images.githubusercontent.com/39388873/88964879-d9bca880-d25e-11ea-8d73-b6ca0a60d376.png)